### DISCUSSION TITLE
Linux computers on meshcentral doesn't appears on Tactical

### DISCUSSION THREAD

**Discussion by mickss41** [amidaware/tacticalrmm#1498] [Q&A]
Hello everyone,

I would like to understand why when I add a computer under linux in Meshcentra this one does not go up in Tactical RMM.

Thank you for your answers.
I wish you a good day

**Comment by dinger1986** [comment]
hello! Are you a sponsor? If not how did you add the agent?

**Reply by mickss41** [reply]
Hello Dinger,

Thanks for your answer :)
No im not, we can install mesh agent without being a sponsor in meshcentral directly.
In Tactical yes it's not possible without sponsor access.

**Reply by dinger1986** [reply]
That's correct and you just answered your own question 😀

**Reply by mickss41** [reply]
So it's not possible to display my linux computers on tactical if im not sponsor ...

**Reply by dinger1986** [reply]
installing via mesh it wont show up in tactical, you can compile your own agents if you dont want to sponsor

**Reply by mickss41** [reply]
Thank you very much Dinger !
I think it's on the FAQ im gonna read how to compile my agents.
